keras/callbacks.py

```diff
@@ -2350,12 +2349,12 @@ class TensorBoard(Callback, version_utils.TensorBoardVersionSelector):
         write_steps_per_second: whether to log the training steps per second
           into Tensorboard. This supports both epoch and batch frequency
           logging.
-        update_freq: **disabled**
-
-          Warning: Batch-level summary writing using `update_freq` is
-          currently unsupported. A suggested workaround is shown in the
-          [TensorBoard Scalars tutorial](https://www.tensorflow.org/tensorboard/scalars_and_keras#batch-level_logging). # pylint: disable=protected-access
-
+        update_freq: `'batch'` or `'epoch'` or integer. When using `'batch'`,
```

Comment:
Can you make it explicit that this only works for a subset of `tf.distribute.Strategy`s? And, when user is using batch or integer, there may be a performance implication and users must be cautious?